### PR TITLE
[README] Table of contents

### PR DESCRIPTION
**Pull request by micpiotrowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```
Comment (mkondratek) [inline README.md:1]: awesome

File: README.md [hunk 2]
```diff
@@ -41,19 +67,15 @@ In that case, just click `Restart IDE` and confirm that action in a message box.
...
 Git Machete IntelliJ Plugin is available under `Git` tool window in the `Git Machete` tab.
 You can also use `Ctrl` + `Alt` + `Shift` + `M` shortcut to open it.
 
 ![](docs/open_git_machete.gif)
-</details>
 
-<details>
-<summary><strong>Discover</strong></summary>
-<br/>
+
```
Comment (mkondratek) [inline README.md:77]: is not a single empty line enough (here and everywhere XD)?

**Review by mkondratek — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (micpiotrowski) [reply README.md:77]: I think two blank lines between sections make this document more clear

File: README.md (hunk 1 above)
Comment (PawelLipski) [reply README.md:1]: Pls add a link to `doctoc` + a line about how to install/run

File: README.md (hunk 2 above)
Comment (mkondratek) [reply README.md:77]: Fine, be consistent then

File: README.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Table of Contents
+
+- [Git Machete IntelliJ Plugin](#git-machete-intellij-plugin)
```
[the remark this answers is not in the document]
Comment (mkondratek) [reply README.md:5]: let's try it

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**